jsbn/README.md

```diff
@@ -0,0 +1,20 @@
+# cljsjs/jsbn
+
+[](dependency)
+```clojure
+[cljsjs/jsbn "0.1.1-1"] ;; latest release
```

Comment:
`+lib-version+` is `0.1.0` but it says `0.1.1` here. (If you don't fix it, I believe @deraen-robot does it automatically.)